src/styles/blocks/card.scss

```
@@ -0,0 +1,108 @@
+@import "./stars.scss";
+@import "./utils/variables";
+@import "./utils/extends";
+
+.products {
+  @extend %column-direction;
+
+  align-items: center;
+  padding: 50px 0 25px;
+
+  &__row {
+    display: flex;
+    margin-bottom: $card-column-gap;
+
+    .card {
+      @extend %column-direction;
+
+      max-width: 200px;
+      padding: 32px 16px 16px;
+      margin-left: $card-row-gap;
+      margin-right: $card-row-gap;
+
+      border: 1px solid #f3f3f3;
+
+      &__img {
+        width: 160px;
+        height: 134px;
+        margin: 0 auto 40px;
+      }
+
+      &__title {
+        margin-bottom: 4px;
+
+        font-size: 12px;
+        font-weight: 500;
+        line-height: 18px;
+        color: $color-black;
+      }
+
+      &__product-code {
+        margin-bottom: 16px;
+
+        font-size: 10px;
+        font-weight: 400;
+        color: $color-gray;
+        line-height: 14px;
+      }
+
+      &__rating {
+        @extend %row-direction;
+
+        justify-content: space-between;
+        align-items: flex-end;
+        margin-bottom: 24px;
+
+        line-height: 14px;
+      }
+
+      &__reviews {
+        font-size: 10px;
+        font-weight: 400;
+        color: $color-black;
+      }
+
+      &__price-title {
+        font-size: 12px;
+        font-weight: 400;
+        color: $color-gray;
+      }
+
+      &__price {
+        @extend %row-direction;
+
+        justify-content: space-between;
+        align-items: flex-end;
+        margin-bottom: 16px;
+
+        line-height: 18px;
+      }
+
+      &__price-uah {
+        font-size: 16px;
+        font-weight: 700;
+        color: $color-black;
+      }
+
+      &__button {
+        padding: 10px 55px 11px;
+
+        font-size: 14px;
+        font-weight: 700;
+        text-transform: uppercase;
+        text-decoration: none;
+        color: #fff;
+
+        border: 1px solid $color-blue;
+        background-color: $color-blue;
+        border-radius: 5px;
+      }
+
+      &__button:hover {
+        color: $color-blue;
+
+        background-color: #fff;
+      }
```

```suggestion
      &__price {
        @extend %row-direction;

        justify-content: space-between;
        align-items: flex-end;
        margin-bottom: 16px;

        line-height: 18px;
        
        &-title {
          font-size: 12px;
          font-weight: 400;
          color: $color-gray;
        }
        
        &-uah {
          font-size: 16px;
          font-weight: 700;
          color: $color-black;
        }
      }

      &__button {
        padding: 10px 55px 11px;

        font-size: 14px;
        font-weight: 700;
        text-transform: uppercase;
        text-decoration: none;
        color: #fff;

        border: 1px solid $color-blue;
        background-color: $color-blue;
        border-radius: 5px;
        
        &:hover {
          color: $color-blue;

          background-color: #fff;
        }
      }
```
Read again about BEM nesting, I also think it's not neccesarry to add an empty line between css properties if there's only two of them.

---

thx